Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:20 as build-stage
+WORKDIR /app
+COPY package*.json ./
+RUN npm install -g @angular/cli
+RUN npm install
+COPY . .
+ARG CONFIG_SERVICE_ENDPOINT
```

Comment:
I think there are mutiple point in that comment

1. I can move the nginx out to a seperate file but for the moment its not needed since it is just the default nginx conf.

2. Regarding the env variable:
Sadly after coding the frontend, I found out that angular does not natively support env variables at runtime. What is required for kubernetes to work?

3. Regarding the dev proxy:
What do I need it for?

Reply:
1. ? The nginx config is already a separate file?
2. You don't use the env variable with angular, you use it with nginx to proxy the API calls. For example, see how the Frontend handles requests to /keycloak
3. https://angular.io/guide/build#proxying-to-a-backend-server